src/oscar/apps/dashboard/ranges/views.py

```diff
@@ -174,11 +233,12 @@ def handle_query_products(self, request, range, form):
                 ", ".join(missing_skus))
         self.check_imported_products_sku_duplicates(request, products)
 
-    def handle_file_products(self, request, range, form):
+    def handle_file_products(self, request, range, form, included=True):
```

Comment:
Again, you could remove included=True here because you can access the type from the `form.cleaned_data["upload_type"]`